openid-4-verifiable-presentations-1_0.md

```diff
@@ -1301,6 +1302,8 @@ The JWE `alg` algorithm used MUST be equal to the `alg` value of the chosen `jwk
 If the selected public key contains a `kid` parameter, the JWE MUST include the same value in the `kid` JWE Header Parameter (as defined in [@!RFC7516, Section 4.1.6]) of the encrypted response. This enables the Verifier to easily identify the specific public key that was used to encrypt the response.
 The JWE `enc` content encryption algorithm used is obtained from the `encrypted_response_enc_values_supported` parameter of client metadata, such as the `client_metadata` request parameter, allowing for the default value of `A128GCM` when not explicitly set.
 
+When the `encryption_details_supported parameter` `client_metadata` parameter is used in the request (see #new_parameters) and requires the Wallet to include a referenced profile in the response, the selected profile can be included in the JWE protected header using the `encryption_details` parameter. When present, the `encryption_details` parameter MUST contain a string of the selected profile.
```

Comment:
```suggestion
When the `encryption_details_supported` parameter `client_metadata` parameter is used in the request (see #new_parameters) and requires the Wallet to include a referenced profile in the response, the selected profile can be included in the JWE protected header using the `encryption_details` parameter. When present, the `encryption_details` parameter MUST contain a string of the selected profile.
```

Reply:
Is this a Verifier metadata parameter that is only used in the request, or is it a new Verifier metadata? In the latter case (e.g., OpenID Federation potentially may want that but may not), we would need to add this to section [11. ](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-11)[Verifier Metadata (Client Metadata)](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#name-verifier-metadata-client-me).

Reply:
> Is this a Verifier metadata parameter that is only used in the request, or is it a new Verifier metadata?

what's the difference to be honest? should we move vp_formats_supported to client-metadata section and just point to client_metadata section from section 11.1?